docs/docs/glossary.md

```diff
@@ -216,6 +220,10 @@ The process of requesting specific data from somewhere. With Gatsby you normally
 
 A code library (written with [JavaScript](#javascript)) for building user interfaces. It’s the framework that [Gatsby](#gatsby) uses to build pages and structure content.
 
+### Remark
+
+A markdown parser to translate [Markdown](#markdown) to other formats like [HTML](#html) or [React](#react) code.
```

Comment:
```suggestion
A parser to translate [Markdown](#markdown) to other formats like [HTML](#html) or [React](#react) code.
```
Prefer to keep the first usage of a term linked & I think "a parser" is clearer anyway?